### PR TITLE
Feat/rewrite error return

### PR DESCRIPTION
**Pull request by LuisOw**
### Testing Instructions

1. Open the PR and review the changes.

### CODE REVIEW

File: pkg/api/handler/sheet_handlers.go [hunk 1]
```diff
@@ -56,62 +59,79 @@ func (sh *SheetHandler) GetSheetHandler(c echo.Context) error {
...
+
+	err = sr.Delete(id)
+	if err != nil {
+		return customError(c, http.StatusInternalServerError, errors.DbError, "delete")
+	}
 
 	return c.NoContent(http.StatusNoContent)
 }
+
+func customError(c echo.Context, https int, message string, args ...any) error {
```
Comment (xsadia) [inline pkg/api/handler/sheet_handlers.go:131]: should this be a exported function inside errors.go?

Comment (LuisOw) [reply pkg/api/handler/sheet_handlers.go:131]: Could be, but I would have to change the return frrom c.JSON to just the error message

Comment (LuisOw) [reply pkg/api/handler/sheet_handlers.go]: Done https://github.com/Roll-Play/roll-play-api/pull/13/commits/a24a4819a27247d92180ebca005e1ea7a8abb2b4

Comment (xsadia) [reply pkg/api/handler/sheet_handlers.go]: no you wouldn't dumb dumb

**Pull request merged**